aiida_vasp/parsers/quantity.py

```diff
@@ -74,8 +74,16 @@ def get_missing_files(self, quantity_name):
     def setup(self):
         """Set the parsable_quantities dictionary based on parsable_items obtained from the FileParsers."""
 
+        # Handle the STDOUT file - the exact file name of STDOUT can be customised by the options
+        stdout_file = self._vasp_parser.node.get_option('scheduler_stdout')
```

Comment:
Should use `VaspCalculation._VASP_OUTPUT` here

Reply:
Indeed.